bin/barnard59.js

```diff
@@ -65,30 +66,36 @@ program
   .option('--pipeline [iri]', 'IRI of the pipeline description')
   .option('--variable <name=value>', 'variable key value pairs separated by comma', parseVariables, new Map())
   .option('-v, --verbose', 'enable diagnostic console output')
-  .action((filename, options = {}) => {
-    let { format, output, pipeline, verbose } = options
+  .option('--debug-buffer', 'enable histogram of buffer usage')
+  .action(async (filename, options = {}) => {
+    try {
+      let { format, output, pipeline, verbose, debugBuffer } = options
 
-    runner.log.enabled = verbose
+      runner.log.enabled = verbose
```

Comment:
```suggestion
      runner.log.enabled = verbose && !debugBuffer
```

Reply:
I know what you mean. Let's see if there are any problems running both at the same time. If there are problems, let's see if we can fix it or throw already an error if both options are given at the same time.

Reply:
Fair enough